### ISSUE TITLE
Add help menu

### ISSUE THREAD

**Issue by FunctionalHacker** [Aanok/jftui#4]
Like the title says, there seems to be no way to go back to the previous menu. Below is an example of me going into "Favorites" on my JF instance:
```
===== Rivendell =====
D 1. Favorites
D 2. Continue Watching
D 3. Next Up
D 4. Latest Unplayed
D 5. User Views
> 1

===== Favorites =====
V 1. Deadpool 2 (2018)
D 2. Stargate SG-1
>
```
I suggest the following menu item to be added:
```
===== Rivendell =====
D 1. Favorites
D 2. Continue Watching
D 3. Next Up
D 4. Latest Unplayed
D 5. User Views
> 1

===== Favorites =====
V 1. Deadpool 2 (2018)
D 2. Stargate SG-1
b - Go back to previous menu
>
```

**Comment by Aanok** [comment]
The command you're looking for is `..` and it is listed in the README file with the rest of the grammar.

I decided against putting a reminder with each menu print because it would only become an annoyance after the first few times you saw it. It would also be asymmetrical with the other commands, as much as one may argue `..` to be the most important.

I will eventually add a `help` command to output general usage information. I think that ought to cover this and other doubts :)

**Comment by FunctionalHacker** [comment]
Ok I like this even better as `..` feels very at home in the terminal.

I agree that a help command would be good to have. I will rename this issue so we can track that here.

**Comment by seniorm0ment** [comment]
I think having a manpage or wiki would be really nice

**Comment by Aanok** [comment]
Kindly implemented by @hstejas in #31 !

And we do have a wiki here on Github with a few details and tips : )
